src/components/TopBar/components/SearchField/SearchField.scss

```diff
@@ -98,6 +98,7 @@ $stacking-order: (
 
   &::placeholder {
     color: var(--p-text, var(--top-bar-color));
+    transition: (duration() - 33ms) color easing() 33ms;
```

Comment:
Super nit but 33ms can be a sass var.

Reply:
I was thinking that as well, but wasn't sure where to put it so it'd be available across multiple partials. Where would it make sense for this Sass var to be stored?

Reply:
If we need a place to share sass variables across subcomponents we occasionally put an `_variables.scss` file in the component folder. For TopBar we've already got  https://github.com/Shopify/polaris-react/blob/master/src/components/TopBar/_variables.scss